Source/Core/Core/IOS/Network/SSL.cpp

```diff
@@ -211,7 +211,7 @@ IPCCommandResult NetSSL::IOCtlV(const IOCtlVRequest& request)
       mbedtls_ssl_conf_cert_profile(&ssl->config, &mbedtls_x509_crt_profile_wii);
       mbedtls_ssl_set_session(&ssl->ctx, &ssl->session);
 
-      if (SConfig::GetInstance().m_SSLVerifyCert && verifyOption)
+      if (SConfig::GetInstance().m_SSLVerifyCert || verifyOption)
```

Comment:
LBTM. If the console requires the certificate not to be verified, it will verify it if ```m_SSLVerifyCert``` is set.

The purpose of that option is to force that insecure behavior when it's set to false (useful for debugging, homebrew, etc.).